plugin/vifm.vim

```diff
@@ -255,6 +255,74 @@ function s:DropGoneBuffers(buffer_snapshot) abort
 	endif
 endfunction
 
+" Opens files after exiting Vifm.  Returns a dict of opened buffer names.
+function! s:OpenFiles(editcmd, flist, opentype) abort
```

Comment:
I had to extract the logic in `HandleRunResults` top open files into its own function. This is to make flow control a little cleaner in `HandleRunResults`.